js/modules/loki_mixpanel.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable class-methods-use-this */
+
+const EventEmitter = require('events');
+const Mixpanel = require('mixpanel');
+// require('setimmediate');
+
+class LokiMixpanelAPI extends EventEmitter {
```

Comment:
Is there a need for deriving EventEmitter?

Reply:
nope, removed.